docs/tasks/administer-cluster/cilium-network-policy.md

```diff
@@ -42,7 +43,7 @@ policies, as well as L7 (e.g., HTTP) security policies using an example applicat
 ## Deploying Cilium for Production Use
 
 For detailed instructions around deploying Cilium for production, see:
-[Cilium Administrator Guide](http://cilium.readthedocs.io/en/latest/admin/) This
+[Cilium Administrator Guide](https://cilium.readthedocs.io/en/latest/install/#kubernetes-installation-guide) This
```

Comment:
if the admin guide is gone, we should rename the link to installation guide.

Reply:
Addressed